clang/lib/CIR/Dialect/Transforms/CIRFlattening.cpp

```diff
@@ -0,0 +1,6 @@
+#include "mlir/Pass/PassManager.h"
+#include "clang/CIR/Dialect/Passes.h"
+
+void mlir::populateCIRFlatteningPasses(mlir::OpPassManager &pm) {
+  pm.addPass(mlir::createStructuredCFGPass());
```

Comment:
the name of this method should be related to the name of the create pass.

Reply:
Overall, I don't see a need for this file or for `populateCIRFlatteningPasses`.

Reply:
ok, removed! and the function renamed and moved into `CIRPasses.cpp`